### PR TITLE
Add AAD hybrid auth workbook 

### PR DESCRIPTION
**Pull request by MichelleSwafford**
This is a hybrid auth workbook that gives customers visibility into add/remove users and groups from staged rollout and also tracks state of sign-ins for users in staged rollout

## PR Checklist

* [ ] Explain your changes, so people looking at the PR know *what* and *why*, the code changes are the *how*.
* [ ] *if updating templates, it is helpful to post a screenshot of what the changes look like.*
* [ ] *if updating templates, ensure that your parameters and steps have meaningful names.*
* [ ] *if adding new templates, ensure that your parameters and columns have display names.*
* [ ] *if adding new templates, or changing items in a gallery, it's helpful to post a screenshot of what the gallery looks like now*

### CODE REVIEW

**Comment by MichelleSwafford** [comment]
This is a hybrid auth workbook that gives customers visibility into add/remove users and groups from staged rollout and also tracks state of sign-ins for users in staged rollout

**Comment by gardnerjr** [comment]
because this is in a new *top level* folder, it doesn't automatically show up as owned by the AAD  workbooks team.  you probably want to move this from
`Workbooks/Azure Active Directory Hybrid Auth Staged Rollout/Hybrid Auth Staged Rollout/StagedRollout.workbooks.txt`

to one folder *inside* of the existing `Azure Active Directory` folder?

`Workbooks/Azure Active Directory/Hybrid Auth Staged Rollout/Hybrid Auth Staged Rollout/StagedRollout.workbooks.txt`

**Comment by gardnerjr** [comment]
Also, it looks like you previously had a PR open to do the same thing, and closed it and made a new one?  you can keep making changes to an existing PR while in progress, you don't need to close the other one and start over (if you do, people won't see the history of previous comments in the PR, etc)

**Comment by MichelleSwafford** [comment]
I think that Hybrid auth should have its own category. I'm planning to add more workbooks under that category like workbooks for PTA.

**Comment by gardnerjr** [comment]
that's fine then, but you'll also need to add a line to the CODEOWNERS file (by the existing ones) that assign code owners to your new top level folder then.  otherwise my team is required to approve things instead of your team.

**Comment by MichelleSwafford** [comment]
Can this be checked-in?

**Comment by gardnerjr** [comment]
has anyone from AAD/your team reviewed this to approve it?  Again, workbooks team can only verify for some things, we don't know how your features work or what to expect inside the template when it runs.  There aren't any screenshots, examples, etc, and we might not have access to the data so if i try to load this template, it might just be a big "no data found" for us.

**Comment by MichelleSwafford** [comment]
This has been reviewed and approved by my team and management. 
It'll be reviewed again by the team once I check it in as preview. 
I'll remove the preview only after approval from the team.

**Comment by gardnerjr** [comment]
@MichelleSwafford  if that is true, someone from your team should literally sign off on it here as well.

ideally, you'd complete your CODEOWNERS changes PR first, which should also enforce that review by someone from that team here as well.

File: Workbooks/Azure Active Directory Hybrid Auth Staged Rollout/Hybrid Auth Staged Rollout/settings.json
Comment (gardnerjr) [inline Workbooks/Azure Active Directory Hybrid Auth Staged Rollout/Hybrid Auth Staged Rollout/settings.json:5]: `"isPreview": true,` lower case `i`, case is sensitive in these properties, as far as i know.

**Review by danielwood95 — APPROVED** [review]
Approving changes. Michelle fixed isPreview and made lowercase

**Comment by gardnerjr** [comment]
can you complete the codeowners PR #1002 first, or make that change here as well?  otherwise, this one still shows up as owned by our team for some amount of time until/if that PR is ever merged.

**Comment by danielwood95** [comment]
> can you complete the codeowners PR #1002 first, or make that change here as well? otherwise, this one still shows up as owned by our team for some amount of time until/if that PR is ever merged.

I've closed PR #1002 since Michelle is already part of @azure-ad-workbooks and doesn't need to be individually added to CODEOWNERS

**Review by danielwood95 — APPROVED** [review]

**Comment by gardnerjr** [comment]
@danielwood95 except that as i keep saying, this PR does not contain CODEOWNERS changes, so without #1002, there's *no ownership of this folder*.  either **this** pr needs codeowners changes or **that** pr needs to be fixed+completed.  also sending mail to clarify.

**Comment by gardnerjr** [comment]
(what's strange is that in one place it says it is waiting for a code owner approver, but Daniel has approved it so the checks here are all green at the bottom)

**Pull request merged**
